utils/format_unifier.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+from pptx import Presentation
+
+
+class FormatUnifier:
+    def extract_text_from_document(self, path_to_file: str) -> bytes:
```

Comment:
Ja bym wolał widzieć `extract_text` metodę, która zwraca od razu gotowy tekst. Nie sądzę by ktoś spoza tej klasy miałby przejmować się odczytywaniem pliku. Raczej ten `FormatUnifier` powinien mieć w sobie klienta Tiki i sam ogarniać to

Reply:
tak